src/librustc_metadata/diagnostics.rs

```diff
@@ -26,6 +26,16 @@ name. Example:
 ```
 "##,
 
+E0455: r##"
+A framework kind link was used and the OS utilized to compile isn't the OSX. Erroneous code example:
```

Comment:
Oh and the sentence sounds strange here as well. Did you mean "A framework link kind..."?